src/pages/using-spark/guides/content-style-guidelines.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Content Style Guidelines
+---
+
+# Content Style Guidelines
+
+UX Writers, designers and developers can use the **Content Style Guidelines** as a guide to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.
+
+### Table Of Contents
+
+- [Capitalization](#capitalization)
+  - [Sentence Case](#sentence-case)
+  - [Start Case](#start-case)
+- [Quotation Marks](#quotation-marks)
+- [Accessibility Guidelines](#accessibility-guidelines)
+
+## Capitalization
+
+Use for visual presentation of pieces of text to make it easy for users to read.
+
+### Start Case
+
+Use this case style to capitalize the first letter of each word, regardless of length or part of speech. When using start case, avoid using end punctuation other than question marks.
+
+#### Usage
+
+Use for:
```

Comment:
We don't need "use for" because the header above it already clearly implies that it's for usage. Let's remove this for every section.